check.sh

```diff
@@ -76,6 +76,18 @@ function findGodot() {
         echo "Found 'godot4.bat' script"
         godotBin="godot4.bat"
 
+    # This should come last: only use this as a last resort as usually `godot`
+    # refers to a Godot 3.x installation.
+    elif command -v godot &>/dev/null; then
+        # Check if `godot` actually is Godot 4.x
+        if godot --version | grep -qE "^4\\."; then
+            echo "Found 'godot' executable with version $(godot --version)"
+            godotBin="godot"
+        else
+            echo "Godot executable found, but it is the incompatible version $(godot --version)"
```

Comment:
A bit clearer, naming the executable "godot" (and also "has" instead of "is):
```suggestion
            echo "Found 'godot' executable, but it has the incompatible version $(godot --version)"
```